### ISSUE TITLE
i10n

### ISSUE THREAD

**Issue by codedust** [publiccodenet/assessment-eligibility#34]
Hi! Is there a possibility to contribute translations of the eligibility assessment?

**Comment by Ainali** [comment]
For now, we don't have the codebase in order for it. However, we have been talking about doing it on one of our codebases just to get some more hands-on experience for it. Which language would you be interested in contributing to?

**Comment by codedust** [comment]
German. Maybe, copying `config.yml` and deploying multiple instances via CI might be the easiest, low-tech solution. 🤔

**Comment by Ainali** [comment]
Good point. Since this site contains just a small amount of text and is not likely to have many changes to it, that might be a way forward.
It might be good for you to know that I'm out on holiday for a couple of weeks and my colleague @ericherman will also be taking some time off, so we are unlikely to have a deep conversation about this before August ends, but we'll keep this issue open.

**Comment by codedust** [comment]
Hi @Ainali! Any news on this topic? I'd be happy to provide a German translation of the config file if that helps. :)

**Comment by ericherman** [comment]
Apologies for the long delay.
We would welcome a German translation, although we have not yet looked into how to support it properly.
We need to investigate what library would be good to use, and having the translated texts would drive that need.
Hopefully, translation support will be straight-forward to add and to contribute upstream.
If you have library recommendations we would happily look at those.
Regardless, if you open a pull-request with the German translations, I'm confident that we will be able to adapt that to whatever format is required in the end.

**Comment by bzg** [comment]
I just added german i18n for the UI here: https://git.sr.ht/~bzg/choices/tree/main/item/src/cljs/choices/i18n.cljs

As for allowing `config.yml` in multiple languages (e.g. `config.de.yml` along  `config.yml`), I will think about it.
